horreum-backend/src/main/java/io/hyperfoil/tools/horreum/svc/Util.java

```diff
@@ -653,7 +656,7 @@ public static Instant toInstant(Object time) {
          try {
             return ZonedDateTime.parse(str, DateTimeFormatter.ISO_DATE_TIME).toInstant();
          } catch (DateTimeParseException e) {
-            log.warn("failed to convert "+time+" to timestamp using "+str,e);
+            log.debug("failed to convert "+time+" to timestamp using "+str);
```

Comment:
> Also changed from warn to debug when logging the failed conversion because failed user conversions are expected rather than something that requires a warning.

I agree

Reply:
I think invalid user input should not be logged by default because I don't think the server logs should be where we want to discover that input. I can certainly see a valid use case to being able to turn on debug to identify invalid user input but users should be able to get a response about their timestamps being invalid without checking the server logs.